web/src/features/map-controls/MapControls.tsx

```diff
@@ -36,13 +36,15 @@ function MobileMapControls() {
   return (
     <div className="absolute right-2 top-2 flex space-x-3 pt-[env(safe-area-inset-top)] sm:hidden">
       <Button
-        className="m-0 bg-white/80 p-3 backdrop-blur-sm dark:bg-gray-800/80"
+        size="lg"
+        type="secondary-elevated"
         aria-label="open info modal"
         onClick={handleOpenInfoModal}
         icon={<HiOutlineInformationCircle size={21} />}
       />
       <Button
-        className="m-0 bg-white/80 p-3 backdrop-blur-sm dark:bg-gray-800/80"
+        size="lg"
+        type="secondary-elevated"
         aria-label="open settings modal"
```

Comment:
Can we keep the reduced opacity and `backdrop-blur-sm` classes here for the nice glass-like visuals?

Reply:
There should be a background-glass variant somewhere that is either done or a work in progress (in the Figma).

Reply:
Yes:)